### PR TITLE
Awards list

### PR DESCRIPTION
**Pull request by martinafm**
on Awards view added a third button "Awards List" navigating to newly created "awards list" view issue #83
https://github.com/zetonteam/zeton_react/issues/83

### CODE REVIEW

**Review by mniegrzybowska — APPROVED** [review]
ok to merge. Great work :)

**Review by miniMax18 — APPROVED** [review]

**Comment by miniMax18** [comment]
Wait a moment, we have a collision in paths

File: src/views/Award/Awards.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          Wystąpił błąd podczas wczytywania danych, proszę odświeżyć stronę.
+        </div>
+      </AwardsContainer>
+    );
+
+  return (
+    <AwardsContainer>
+      <div>
+        {awards?.map((award) => (
+          <li key={award.id}>
```
Comment (KGrzeg) [inline src/views/Award/Awards.tsx]: `<li>` elements should be alwayes wrapped by `<ol>` or `<ul>` HTML element.
You could also just pass the `awards` list as a prop to the `<AwardsContainer>` and then render the list inside the component's code.

**Review by KGrzeg — COMMENTED** [review]
Great work!

**Comment by mniegrzybowska** [comment]
> Wait a moment, we have a collision in paths

@miniMax18 what kind of collision? In routes or backend?

**Review by miniMax18 — APPROVED** [review]
Up, @mniegrzybowska this problem was on my site|side. 
Congratulations 👍 
list_awards

File: src/views/Award/Awards.tsx (hunk 1 above)
Comment (martinafm) [reply src/views/Award/Awards.tsx]: Good point, I implemented this into the code!

File: src/views/Award/types.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export type AwardObject = {
```
Comment (mniegrzybowska) [inline src/views/Award/types.ts]: can we name it Award?

File: src/views/Award/Awards.tsx (hunk 1 above)
Comment (KGrzeg) [reply src/views/Award/Awards.tsx]: If there is no awards, we're getting `<ol><div> Brak nagród. </div></ol>`, which's not perfect but for me it's good enough.

Great job!

File: src/views/Award/types.ts (hunk 2 above)
Comment (martinafm) [reply src/views/Award/types.ts]: Renamed ;) done

**Pull request merged**